src/org/loklak/data/DAOWrapper.java

```diff
@@ -0,0 +1,127 @@
+package org.loklak.data;
+
+/**
+ * Created by Deng Yiping on 16-7-31.
+ */
+
+import org.loklak.objects.MessageEntry;
+import org.loklak.objects.QueryEntry;
+import org.loklak.objects.SourceType;
+import org.loklak.objects.UserEntry;
+
+import java.net.MalformedURLException;
+import java.util.Date;
+
+/**
+ * The json below is the minimum json
+ * {
+ "statuses": [
+ {
+ "id_str": "yourmessageid_1234",
+ "screen_name": "testuser",
+ "created_at": "2016-07-22T07:53:24.000Z",
+ "text": "The rain is spain stays always in the plain",
+ "source_type": "GENERIC",
+ "place_name": "Georgia, USA",
+ "location_point": [3.058579854228782,50.63296878274201],
+ "location_radius": 0,
+ "user": {
+ "user_id": "youruserid_5678",
+ "name": "Mr. Bob",
+ }
+ }
+ ]
+ }
+ */
```

Comment:
@DengYiping So as all scrapers have different formats, how should we go about pushing them into the index using the builder ? Does, everything go into "text" (considering the fact we have more fields than the Twitter scarper) ?